src/file_detection.rs

```diff
@@ -3,12 +3,14 @@ use std::fs;
 use std::path::Path;
 
 use crate::constants;
+use crate::core::args::Flags;
+use crate::helper::Die;
 
 #[cfg(target_os = "windows")]
 fn is_hidden_extra(path: &Path) -> bool {
   use std::os::windows::fs::MetadataExt;
 
-  let file_metadata = fs::metadata(path).expect("Failed getting metadata");
+  let file_metadata = fs::metadata(path).die("Failed getting metadata");
```

Comment:
Isn't `flags` supposed to be passed to `die` here?

Reply:
You're completely right, I wonder why the compiler didn't say anything about it.

Reply:
I just noticed, I am working on Linux and that function is only for Windows, therefore the compiler just omitted it.